containers/Makefile

```diff
@@ -68,6 +68,11 @@ postgresql-clean:
 concent-api-clean:
 	docker rmi $(IMAGE_PREFIX)concent-api:$(IMAGE_VERSION) --force
 
+build/concent-api/RELEASE-VERSION: build/nginx-proxy/concent-api-assets.tar
+	build/virtualenv/bin/python3                       \
+	"build/virtualenv/src/golem-messages/version.py"
+	mv RELEASE-VERSION "build/concent-api"
```

Comment:
There should be a separate target for creating the virtualenv. Right now creating the virtualenv is only a side-effect in the step that produces `concent-api-assets.tar` and you can't count that it will always stay like this. If both need the virtualenv, it should be created explicitly.